logicrunner/preprocessor/main_test.go

```diff
@@ -597,47 +597,6 @@ func ( A ) Get() error {
 	s.NotContains(bufWrapper.String(), `"some/test/import/path"`)
 }
 
-func (s *PreprocessorSuite) TestImportsFromContractUseForReturnValue() {
```

Comment:
I don't understand why you deleted this test. Can you explain?

Reply:
We're importing packages needed for declaration of contract return values, now. That test tests that we won't import that packages.

Reply:
Nope, that test was checking that we deal correctly with return values of type declared in an external package. This is what it was testing. If you changed the way it works then you should change the test. Don't delete it.

Reply:
fixed